madgraph/iolibs/export_v4.py

```diff
@@ -6941,10 +6949,8 @@ def copy_standard_file(self):
         file = open(os.path.join(MG5DIR,\
                               'models/template_files/fortran/rw_para.f')).read()
 
-        includes=["include \'../vector.inc\'",
-                  "include \'coupl.inc\'",
-                  "include \'input.inc\'",
-                  "include \'model_functions.inc\'"]
+        includes=["include \'coupl.inc\'","include \'input.inc\'",
```

Comment:
Here I'm confused why vector.inc is not include here. (or why it is needed above)